composer/loggers/mlflow_logger.py

```diff
@@ -346,6 +346,55 @@ def log_model(self, flavor: Literal['transformers'], **kwargs):
             else:
                 raise NotImplementedError(f'flavor {flavor} not supported.')
 
+    def create_model_version(
```

Comment:
where do we plan to call this function?

Reply:
https://github.com/mosaicml/llm-foundry/pull/953